snuba/admin/views.py

```diff
@@ -6,3 +6,22 @@
 @application.route("/")
 def root() -> Response:
     return application.send_static_file("index.html")
+
+
+@application.route("/test-slack")
+def test_slack() -> Response:
+    from typing import Dict, Union
+
+    from snuba.admin.slack.client import runtime_config_slack_client
+
+    data: Dict[str, Union[str, float, int]] = {
+        "option": "enable_events_read_only_table",
+        "old": 0,
+        "new": 1,
+    }
+
+    runtime_config_slack_client.notify(
```

Comment:
@nikhars I left this up just to show an example of what I was thinking we could do. I think we probs should change the action to be an enum but yeah anyway some sort of action arg